sdk/javascript/package-lock.json

```diff
@@ -4514,9 +4514,9 @@
       }
```

Comment:
package.json needs to be updated too. usually i update that first and then do npm install to let it regenerate package-lock. there is also npm outdated (which is command i think dependabot is running).